docs/content/java-support.md

```diff
@@ -0,0 +1,20 @@
+## Java Language Support
+
+From version 0.0.32 we provide [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [VS Code Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) as dependencies.
+
+With [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java), you will benefit from Java Language completion on standalone files. An invisible project is created with a default classpath.
+
+The command `Refresh local Java classpath for Camel K standalone file based on current editor. Available with kamel 1.4+.` refreshes specific dependencies for the classpath of the opened Integration, such as dependencies declared as part of the modeline of the file. When using a modeline to configure such dependencies, you gain a [CodeLens](https://code.visualstudio.com/blogs/2017/02/12/code-lens-roundup) link (`Refresh classpath dependencies`) at the top of the editor to trigger the refresh more easily.
+
+Be aware of the following **limitations**:
+
+- It requires Camel K 1.4.0
+- If mistakenly called with Camel K 1.3.2-, need to restart VS Code for basic dependencies to be available again.
+- It supports modeline dependencies notation from local build. See [apache/camel-k#2213](https://github.com/apache/camel-k/issues/2213)
+- A single classpath is provided. It means that refresh command needs to be called when switching between Integration file written in Java that does not have the same dependencies.
+- There is no progress indicator. Please be patient. The first time may take several minutes on a slow network.
+  
+[VS Code Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) is provided to benefit from Java debug on standalone files. To leverage it, you need to start an integration, then there are 2 solutions:
```

Comment:
WDYT, use the name as in the marketplace:
- VS Code Debugger for Java -> Debugger for Java